File: Lógica de Programação Java e C#/4 - Controle de Fluxo/C#/Exercícios Complementares/JogoSomaImpar.cs
class JogoSomaImpar
{
    static void Main()
    {
        int soma = 0, cont = 0;
        System.Random gerador = new System.Random();
        for (int a = 1; a <= 10; a++)
        {
            int valor = (int) (gerador.NextDouble() * 6 + 1);
            System.Console.WriteLine(valor);
            if (valor == 6)
            {
                cont++;
                if (cont >= 2)
                {
                    System.Console.WriteLine("Você já perdeu o número 6 apareceu duas vezes");
                    break;
                }
            }
            soma += valor;
        }
        if (soma % 2 != 0 && cont<2)
        {
            System.Console.WriteLine("Você ganhou!! a soma é ímpar = "+soma);
        }
        if(soma % 2 == 0 && cont<2)
        {
            System.Console.WriteLine("Você perdeu!! A soma é par "+soma);
        }
            
    }
}

File: Lógica de Programação Java e C#/4 - Controle de Fluxo/Java/Exercícios Complementares/JogoSomaImpar.java
class JogoSomaImpar{
	public static void main(String [] args){
		int soma=0,contador=0;
			for(int jogada=1;jogada<=10;jogada++){
				int dado = (int)(Math.random() * 6 + 1);
				System.out.println(dado);
					if(dado!=1){
						soma +=dado;
					}	
					if(dado == 6){
						contador++;
						if(contador>=2){
							break;
						}
					}	
			}
			
			if(soma%2!=0 && contador < 2){
				System.out.println("Você ganhou a soma é ímpar "+soma);	
			}
			if(soma%2==0 && contador<2){
				System.out.println("Você perdeu!! A Soma é par "+soma);
			}if(contador>=2){
				System.out.println("Você perdeu!! o número 6 aparece duas vezes");
			}
	}
}

		